content/Arduino Cloud/General Cloud/Wi-Fi-troubleshooting.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Wi-Fi troubleshooting"
+---
+
+This guide is intended to help you troubleshoot common Wi-Fi connectivity issues with your devices. Follow the steps below to identify and resolve potential issues.
+
+## 1. Frequency Bands: 2.4 GHz vs 5 GHz
+
+Learn more about it [here](https://support.arduino.cc/hc/en-us/articles/14144656051740-Supported-Wi-Fi-bands)
+
+## 2. Avoid common network names
+
+Avoid using generic names like "default" or "home" to prevent interference and confusion with neighboring networks.
+
+## 3. Reset Network Credentials
+
+### Verifying or changing network credentials in Arduino Cloud
+
+If the Wi-Fi connection is not working, you might need to reset the [network credentials](https://support.arduino.cc/hc/en-us/articles/360016495559-Add-and-connect-a-device-to-Arduino-Cloud#:~:text=Click%20the%20Network,adding%20the%20device) for your Thing in Arduino Cloud.
+
+1. Navigate to your [Thing](https://app.arduino.cc/things) that is experiencing Wi-Fi issues.
+
+1. Under **Network**, click the button labeled "Change"
+
+1. Re-enter your Wi-Fi credentials with your Wi-Fi Name (SSID) and Password (Encryption Key). Ensure there are no typos and the credentials match those set on your router.
+
+1. Open the sketch and click the **Upload** button to properly apply the new network settings to your device.
+
+### Veryfing or changing the credentials defined in the sketch
+
+To verify or change the network credentials directly defined in a sketch, whether the sketch was created in Arduino Cloud or or IDE, follow these steps:
+
+1. Locate the section in your Arduino sketch where the network credentials are defined. This is typically found in the main sketch file or inside an `arduino_secrets.h` file.
+
+1. Check the variables for SSID and password to ensure they match your network credentials. You may also need to modify key index if you're using a WEP encrypted network.
+
+1. After making any necessary changes, save the sketch.
+
+1. Upload the modified sketch by clicking the Upload button in the Arduino IDE or Cloud Editor.
+
+Additionally, you can check the console (serial) output for error messages. This can provide insight into what might be going wrong during the connection process.
+
+## 4. Check your Power Supply
+
+Wi-Fi connectivity problems are often caused by insufficient power. Since Wi-Fi usage demands more power than normal operations, ensure your device is adequately powered.
```

Comment:
```suggestion
Wi-Fi connectivity problems are often caused by insufficient power. Since Wi-Fi usage demands more power than other operations, ensure your device is adequately powered by:
```